.github/workflows/e2e-test.yml

```diff
@@ -36,24 +36,22 @@ jobs:
       fail-fast:     false
       matrix:
         # Min WooCommerce | L-2 | L(atest) | beta
-        woocommerce:   [ '6.0.1', '6.5.1', 'latest', 'beta' ]
+        woocommerce:   [ '6.2.2', '6.6.0', 'latest', 'beta' ]
         wordpress:     [ 'latest', 'nightly' ]
         php:           [ '7.4' ]
         test_groups:   [ 'wcpay', 'subscriptions', 'blocks' ]
         test_branches: [ 'merchant', 'shopper' ]
         exclude:
           - test_groups: 'blocks'
             test_branches: 'merchant'
-          - woocommerce: '6.0.1'
+          - woocommerce: '6.2.2'
             test_groups: 'blocks'
             test_branches: 'shopper'
-          - woocommerce: '6.5.1'
+          - woocommerce: '6.2.2'
```

Comment:
Should there be the following duplication here:
```yml
- woocommerce: '6.2.2'
  test_groups: 'blocks'
  test_branches: 'shopper'
- woocommerce: '6.2.2'
  test_groups: 'blocks'
  test_branches: 'shopper'
```

It seems like we may want the following instead:
```yml
- woocommerce: '6.2.2'
  test_groups: 'blocks'
  test_branches: 'shopper'
- woocommerce: '6.6.0'
  test_groups: 'blocks'
  test_branches: 'shopper'
```

Reply:
I removed the duplicate 6.2.2 in [18034d1](https://github.com/Automattic/woocommerce-payments/pull/4524/commits/18034d1e3a111a2f209be482d34c442bf813eb97) based on this: #4172